hd/lang/lex_utf8.txt

```diff
@@ -10,20 +10,6 @@ fr: Tapez à l'identique le nom de la généalogie que vous voulez utiliser :
 en: list of available genealogies (use gwsetup)
 fr: listes des généalogies disponibles (utilise gwsetup)
 
-    about (short date)
```

Comment:
I told you before, do a separate PR for that. The rule is: if this is not related to what you are implemented, do not put it in the PR.

It pollutes the diff to read. How should I suppose to know that this is not related to this PR ?